config/settings.yml

```diff
@@ -22,3 +22,4 @@
       :event_catcher_nuage_network:
         :topics:
           - topic/CNAMessages
+        :poll: 1.seconds
```

Comment:
Please add empty line at the end of the file

Reply:
Hmmm, not sure. Looking at the changes, I don't think I actually removed it. I checkout `master` and there also is no newline (nor is there any in OpenStack provider).

Reply:
Wanna bet for a burger? 😉 

There is empty trailing line both in Amazon and OpenStack provider's settings.yaml, only GitHub won't render it. Instead, if there is no empty line at the end of file, GitHub would render the ugly red sign, like in this case.

![capture](https://user-images.githubusercontent.com/8102426/34213290-061c1db6-e59f-11e7-80c6-867dae6f3abe.PNG)

Reply:
Will this `:poll` setting be read or will the one in the manageiq repo be read instead? See  [here](https://github.com/ManageIQ/manageiq/blob/master/config/settings.yml#L1070). The reason i ask is that the `sleep_poll_normal` looks for a `workers` section of the settings. 
See:
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_worker/runner.rb#L448..#L450
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_worker/runner.rb#L302
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_worker.rb#L221..#L223